hardcoded.php

```diff
@@ -0,0 +1,52 @@
+<?php
+
+// Used to generate data for consts.php INSTALLER_TO_REPO_MINOR_VERSIONS
+// Copy .cow.pat.json from the current release into the directory this script is being run from (file is .gitignored)
+
+include 'consts.php';
+
+$filename = '.cow.pat.json';
+if (!file_exists($filename)) {
+    throw new \RuntimeException("Copy $filename into the directory this is being called from");
+}
+
+$versions = [];
+
+function repoName(string $name)
+{
+    $prefix = '';
+    if (!preg_match('#^(cwp/)#', $name) &&
+        !preg_match('#(/recipe-|/silverstripe-|/comment-notifications)#', $name)
+    ) {
+        $prefix = 'silverstripe-';
+    }
+    if (preg_match('#(agency-extensions|starter-theme|watea-theme)#', $name)) {
+        $prefix = 'cwp-';
+    }
+    $name = str_replace('agency-extensions', 'agencyextensions', $name);
```

Comment:
Do this inside the above conditional so we're protecting against possible future `something-agency-extensions` or `agency-extensions-something`

Reply:
this is just not likely